src/test/php/PDepend/AbstractTest.php

```diff
@@ -118,6 +122,26 @@ protected function tearDown()
         parent::tearDown();
     }
 
+    /**
+     * Override to run the test and assert its state.
+     *
+     * @return mixed
+     *
+     * @throws Exception|PHPUnit_Framework_Exception
+     */
+    protected function runTest()
+    {
+        $inputReflector = new ReflectionProperty('PHPUnit_Framework_TestCase', 'dependencyInput');
+        $inputReflector->setAccessible(true);
+        $input = $inputReflector->getValue($this);
+
+        if (!empty($input)) {
+            $inputReflector->setValue($this, array_values($input));
+        }
+
+        return parent::runTest();
```

Comment:
`runTtest()` returns `void
````suggestion
        parent::runTest();
```

Reply:
Not correct, a test can return a result (see `return $testResult;` in **src/Framework/TestCase.php**) that can be ussed by an other test that depends on it.